conman/routes/tests/test_handlers.py

```diff
@@ -34,6 +20,19 @@ def test_init(self):
         self.assertEqual(handler.route, route)
 
 
+class BaseHandlerCheckTest(TestCase):
+    """Test BaseHandler.check()."""
+    def test_classmethod(self):
+        """Ensure check() is a classmethod."""
+        self.assertIsInstance(vars(BaseHandler)['check'], classmethod)
```

Comment:
Oh, I missed that one, sorry. I'll have a look (but I suspect it'll have to be the comment).